### DISCUSSION TITLE
Objets de loot

### DISCUSSION THREAD

**Discussion by ElethornGM** [fibreville/minotaure_php#69] [Boite à idées]
Idée de dev : Possibilité de retirer l'objet d'un personnage (en cliquant sur le personnage par exemple)

Question : Combien d'objets un personnage peut-il posséder actuellement ? y a-t-il une limite ?

**Comment by uurshin** [comment]
Il n'y a pas de limite.
Pour info, quand on ajoute un objet de loot actuellement cela a deux effets pour chaque perso concerné : 
- ajouter une entrée dans la table "loot" avec l'id du personnage + la description de l'objet (nom + effet)
- augmenter la stat concernée du personnage si nécessaire

C'est grosso modo le fonctionnement du loot de la V1. La seule véritable modification qui a été apporté est l'ajout de l'effet entre parenthèse. Cela a pour conséquence d'empêcher d'enlever le loot car une fois la carac augmentée, cet augmentation est totalement séparée de la possession du loot.

D'un point de vue technique, pour les personnes qui souhaiteraient contribuer, il faudrait ajouter dans la table loot deux colonnes, une pour la carac affectée et une autre pour le modificateur associé. Une fois que ce sera fait, on pourra alors envisager de pouvoir enlever le loot d'un personnage.

Concernant les actions individuelles sur chaque personnage, ça fait partie d'un ensemble de fonctionnalité individuelles qu'on pourrait envisager d'ici quelques versions (2.8 peut être ?)
